src/encoded/static/components/dataset.js

```diff
@@ -1744,8 +1744,8 @@ const differentiationTableColumnsWithTime = {
             );
         },
         objSorter: (a, b) => {
-            const aTime = computeDifferentiation(a).split(' ')[0];
-            const bTime = computeDifferentiation(b).split(' ')[0];
+            const aTime = computeDifferentiation(a) ? computeDifferentiation(a).split(' ')[0] : 0;
```

Comment:
You could of had done -

```
const aTime = computeDifferentiation(a)?.split(' ')[0] || 0;
```

Reply:
Oh, I like that better, it's more concise